app/routes/admin.py

```diff
@@ -31,3 +36,38 @@ def delete_answer(id):
         return redirect(f"/post/{post_id}")
     else:
         return abort(400)
+
+
+@is_admin
+@csrf_protected
+@server.route("/admin/merge_users", methods=["POST"])
+def merge_users():
+    source_ids = request.form['source_ids']
+    target_id = request.form['target_id']
+    try:
+        admin_controller.merge_users(source_ids, target_id)
+        return render_json({'source_ids': source_ids, 'target_id': target_id, 'merge_users': True})
+    except:
```

Comment:
might be better to detect the type of exception (ValueError) that way the other exceptions can be caught and given to bugsnag